cli/azd/test/samples/resourcegroups/infra/main.parameters.json

```diff
@@ -0,0 +1,22 @@
+{
+  "$schema": "https://schema.management.azure.com/schemas/2019-04-01/deploymentParameters.json#",
+  "contentVersion": "1.0.0.0",
+  "parameters": {
+    "name": {
+      "value": "${AZURE_ENV_NAME}"
+    },
+    "location": {
+      "value": "${AZURE_LOCATION}"
+    },
+    "resourceGroupName": {
+      "value": "${TEST_RESOURCE_GROUP_NAME}"
+    },
+    "includeEnvNameTag": {
+      "value": "${TEST_INCLUDE_ENV_NAME_TAG}"
+    }
+    ,
+    "createMultipleResourceGroups": {
+      "value": "${TEST_CREATE_MULTIPLE_RESOURCE_GROUPS}"
+    }
```

Comment:
I'm assuming these are added here just to support better test cases and not something we would propose in our templates?

Reply:
Correct, test helpers only.